### DISCUSSION TITLE
[Feature] iOS Live Activity Support

### DISCUSSION THREAD

**Discussion by NeodymiumPhish** [immich-app/immich#13032] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Instead of updating the app notification every time the upload percent changes, I think it would be a much better experience to use the iOS Live Activity feature instead. I think this would also help keep the upload running, since the live activity (unless I’m mistaken) is a way to keep the app active while the LA is running. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8914

**Reply by NeodymiumPhish** [reply]
My mistake, sorry! I searched for live activity and didn’t find it, but my filters must’ve been excluding it
